### DISCUSSION TITLE
Gitter <-> Telegram link may be down

### DISCUSSION THREAD

**Discussion by VisualMelon** [AvaloniaUI/Avalonia#9050] [General]
The avalonia gitter has been very quiet for the past few days: looks like it's not seeing events from telegram (and I assume vice versa, but I can't check).

Sorry, couldn't think of a better place to raise this that doesn't require telegram.

**Comment by timunie** [comment]
It's fixed now, thank you for letting us know. 👍 

Anyway, telegram is the preffered channel if possible as it is the most stable

**Reply by VisualMelon** [reply]
I think it's died again :(

**Reply by furai** [reply]
Are there any community driven IRC channels?
